### PR TITLE
AAE-23121 Refactor Query and Audit Consumer into separate starter modules

### PR DESCRIPTION
**Pull request by igdianov**
The codebase has been refactored to achieve the separation of concerns between Query Rest Apis and Event Consumers modules to be able to build and run 3 flavors of Spring Boot Applications packaged in a single Query Service Docker image:

- Query Starter -> All-In-One Rest and Events Consumer
- Query Rest - Rest and GraphQl Apis
- Query Consumer - Audit and Query Events Consumer

https://hyland.atlassian.net/browse/AAE-23121

### CODE REVIEW

**Comment by atchertchian** [comment]
Hello @igdianov please rebase over https://github.com/Activiti/activiti-cloud/pull/1516 once it's merged if you have more changes coming (not needed if you merge right away), thanks!

File: activiti-cloud-audit-service/activiti-cloud-services-audit-consumer/pom.xml
Comment (erdemedeiros) [inline activiti-cloud-audit-service/activiti-cloud-services-audit-consumer/pom.xml]: Maybe better to remove this line as `compile` is the default scope

File: activiti-cloud-audit-service/activiti-cloud-starter-audit-consumer/pom.xml
Comment (erdemedeiros) [inline activiti-cloud-audit-service/activiti-cloud-starter-audit-consumer/pom.xml:44]: I was wondering if we could remove the dependency to `spring-boot-starter-web` here.

File: activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/pom.xml
Comment (erdemedeiros) [inline activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/pom.xml]: Default scope: can be omitted.

File: activiti-cloud-query-service/activiti-cloud-starter-query-consumer/pom.xml
Comment (erdemedeiros) [inline activiti-cloud-query-service/activiti-cloud-starter-query-consumer/pom.xml:42]: Could this dependency be removed?

**Review by erdemedeiros — APPROVED** [review]
I added some questions. LGTM, otherwise!

File: activiti-cloud-audit-service/activiti-cloud-starter-audit-consumer/pom.xml
Comment (igdianov) [reply activiti-cloud-audit-service/activiti-cloud-starter-audit-consumer/pom.xml:44]: We need it for actuator support

**Pull request merged**
